src/WebJobs.Script.WebHost/Startup.cs

```diff
@@ -41,6 +41,7 @@ public void Configure(IApplicationBuilder app, IApplicationLifetime applicationL
             {
                 loggerFactory.AddConsole(LogLevel.Trace, true);
                 app.UseDeveloperExceptionPage();
+                Environment.SetEnvironmentVariable("FUNCTIONS_WORKER_RUNTIME", "any");
```

Comment:
@fabiocav - Added environment variable here to help with F5 scenario.  Will address moving this out to IScriptWebHostEnvironment as part of https://github.com/Azure/azure-functions-host/issues/3278

Reply:
I think we can remove it for now. We just need to document it.